internal/install/nerdgraph_client.go

```diff
@@ -0,0 +1,5 @@
+package install
```

Comment:
An interface that matches the nerdgraph client exported from `newrelic-client-go` to facilitate unit testing.